examples/tactile_pose_estimation.py

```diff
@@ -281,18 +275,11 @@ def run_learning_loop(cfg):
 
     outer_optim = optim.Adam(learnable_params, lr=learning_rate)
     batch_size = cfg.train.batch_size
-    num_batches = cfg.train.num_batches
-    measurements = [
-        (
-            dataset["img_feats"][0:time_steps].unsqueeze(0).repeat(batch_size, 1, 1),
-            dataset["eff_poses"][0:time_steps].unsqueeze(0).repeat(batch_size, 1, 1),
-            dataset["obj_poses"][0:time_steps].unsqueeze(0).repeat(batch_size, 1, 1),
-        )
-        for _ in range(num_batches)
-    ]
-
-    obj_poses_gt = dataset["obj_poses"][0:time_steps, :].clone().requires_grad_(True)
-    eff_poses_gt = dataset["eff_poses"][0:time_steps, :].clone().requires_grad_(True)
+    measurements = dataset.get_measurements(
+        cfg.train.batch_size, cfg.train.num_batches, time_steps
+    )
+    obj_poses_gt = dataset.obj_poses[0:time_steps, :].clone().requires_grad_(True)
```

Comment:
@psodhi Another question. Is setting `requires_grad_(True)` necessary here?

Reply:
I don't think it is necessary. We can remove it.